experimental/bits/simd.h

```diff
@@ -178,7 +184,7 @@ using __value_type_or_identity_t
 // }}}
 // __is_vectorizable {{{
 template <typename _Tp>
-struct __is_vectorizable : public std::is_arithmetic<_Tp>
+struct __is_vectorizable : public std::is_arithmetic<std::remove_reference_t<_Tp>>
```

Comment:
why is this needed? `_Tp` should never be a reference. And references are not vectorizable. (Pointers might be - needs a proposal)

Reply:
clang by some reason returns long long&& out of operator[] in my example:
https://godbolt.org/z/WJN7_M

Reply:
Then whatever calls `__is_vectorizable<decltype(x[0])>` is incorrect.

Reply:
Yes, thus a number of remove_references will be required to workaround that thing across other places.